src/app/pages/common/entity/entity-form/validators/password-validation/password-validation.spec.ts

```diff
@@ -0,0 +1,103 @@
+import { TestBed } from '@angular/core/testing';
+import {
+  AbstractControl,
+  FormBuilder, FormGroup, FormsModule,
+} from '@angular/forms';
+import { doesNotEqualValidator } from 'app/pages/common/entity/entity-form/validators/password-validation/password-validation';
+import { matchOtherValidator } from './password-validation';
+
+describe('PasswordValidation', () => {
+  describe('matchOtherValidator', () => {
+    let thisControl: AbstractControl;
+    let otherControl: AbstractControl;
+    let form: FormGroup;
+
+    beforeEach(() => {
+      TestBed.configureTestingModule({
+        imports: [FormsModule, FormBuilder],
+        declarations: [matchOtherValidator],
+      });
+
+      const formBuilder = new FormBuilder();
+      form = formBuilder.group({
+        this: ['', [matchOtherValidator('other')]],
+        other: ['', []],
+      });
+
+      thisControl = form.get('this');
+      otherControl = form.get('other');
+    });
+
+    it('should have parent', () => {
+      expect(thisControl.parent).toBeTruthy();
+    });
```

Comment:
Doesn't look like this case actually tests anything in password-validation.
You can change it to 'should return null when control does not have a parent' or you can just remove this case.